components/layout_2020/table/construct.rs

```diff
@@ -159,3 +305,41 @@ where
         // do something?
     }
 }
+
+impl<'a, 'builder, 'dom, Node> TableRowBuilder<'a, 'builder, Node>
+where
+    Node: NodeExt<'dom>,
+{
+    fn consume_rowspans(&mut self) {
+        loop {
+            if let Some(span) = self.builder.incoming_rowspans.get_mut(self.current_x) {
+                if *span != 0 {
```

Comment:
nit: could be `if *span == 0 { break; }` and then avoid some rightward drift (and would also be a bit clearer IMO)

Reply:
In later code that branch also gets complicated